arch/arm/src/common/Toolchain.defs

```diff
@@ -232,9 +232,11 @@ else
   # Wrong warning array subscript [0] is outside array bounds:
   # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105523
 
-  GCCVER = $(shell $(CC) --version | grep gcc | sed -r 's/.* ([0-9]+\.[0-9]+\.[0-9]+).*/\1/')
+  ifeq ($(GCCVER),)
+    export GCCVER := $(shell $(CC) --version | grep gcc | sed -r 's/.* ([0-9]+\.[0-9]+).*/\1/')
```

Comment:
Why export is needed?

Reply:
Avoid calling multiple shells repeatedly, it is enough to obtain the gcc version once